src/main/scala/com/greencatsoft/angularjs/core/Injector.scala

```diff
@@ -0,0 +1,8 @@
+package com.greencatsoft.angularjs.core
+
+import scala.scalajs.js
+
+@js.native
```

Comment:
From the documentation, it seems it can be injected as `$injector` itself. Probably we need to put `@injectable("$injector")` here so it can be used in both ways?